experimental/rsocket/OldNewBridge.h

```diff
@@ -6,15 +6,19 @@
 
 #include <folly/ExceptionWrapper.h>
 
-#include "reactivestreams/ReactiveStreams.h"
+// using the "v" Flowable types
+// since they don't have ReactiveStreams right now
+#include "yarpl/v/Subscriber.h"
+#include "yarpl/v/Subscription.h"
+
 #include "src/Payload.h"
 #include "src/ReactiveStreamsCompat.h"
 
 namespace rsocket {
 
 ////////////////////////////////////////////////////////////////////////////////
 
-class NewToOldSubscription : public reactivestreams_yarpl::Subscription {
```

Comment:
Yes. There's some pain there since RS streams subscribe call has the wrong signature: subscribe(Subscription*). There are no decent ways to handle this case: every subclass ends up needing to implement this, or incur a warning for hidden superclass methods. (Which we can disable, but that seemed bad to me.)